fog/view/server/src/bin/main.rs

```diff
@@ -60,15 +58,15 @@ fn main() {
 
     let ias_client = Client::new(&config.ias_api_key).expect("Could not create IAS client");
 
+    let config::ShardingStrategy::Epoch(sharding_strategy) = config.clone().sharding_strategy;
+
     let mut server = ViewServer::new(
         config.clone(),
         sgx_enclave,
         recovery_db,
         ias_client,
         SystemTimeProvider::default(),
-        // TODO: Change the EpochShardingStrategy to incorporate config values that specify
-        //  start and end block indices. See PR #2352
-        EpochShardingStrategy::default(),
+        sharding_strategy,
```

Comment:
A couple of things:
* The config is being cloned, when it seems like only the sharding strategy needs to be cloned
* There is currently only one strategy, but it seems off to be forcing this to be that one all the time. 
```suggestion
    let mut server = ViewServer::new(
        config.clone(),
        sgx_enclave,
        recovery_db,
        ias_client,
        SystemTimeProvider::default(),
        config.sharding_strategy.clone(),
```

Reply:
Changed the clone. And yes there's one sharding strategy right now, EpochShardingStrategy, and the default version of it is actually what Fog View does currently.